tests/cmake/syscalls/main.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <zephyr/syscall.h>
+
+// fixed in commit 54e369998747ccb89c2e563c4167b4a90c8670b2
```

Comment:
be careful with commit references like this because the SHA will change when the PR is merge (rebased) on main.
This means that in the future then this sha can no longer be retrieved.